mmcv/ops/csrc/onnxruntime/cpu/gridSample.cpp

```diff
@@ -0,0 +1,310 @@
+#include <cmath>
```

Comment:
Is the code modified from PyTorch's source code? It would be better to add some comments like https://github.com/open-mmlab/mmcv/blob/9d80f56ae8c416556f08d0e04d03785ac036c9c7/mmcv/ops/csrc/deform_conv_cuda_kernel.cuh#L63 in front of this file.